docs/operations.rst

```diff
@@ -51,3 +51,39 @@ The important bits here are:
 Backups
 *******
 
+Performing regular backups of the state store is an important operational component for any production deployment of ``alchemiscale``.
+To do this, **first shut down the ``neo4j`` service so that no database processes are currently running**.
```

Comment:
Do we want to add the example docker compose command here to do that?

Reply:
Yes! I'm not exactly sure what the best practice is here, and couldn't find it readily in the `docker-compose` docs. Can you point me to it?

Reply:
`docker compose stop neo4j`

That should stop the neo4j service and everything that it depends on. But maybe that can just be an exercise for the reader. `docker compose start` Will then bring everything back up. This might be too much in the weeds...

Reply:
Yeah, I think I'll leave it as-stated; we state below that the instructions to follow assume a Docker-based deployment, but that the same principle applies for other deployment-types (e.g. kubernetes, or even manually-installed Neo4j + systemd).

Appreciate this though; I'll try out the `docker compose stop` commands next time I do operations like this. :pray: